gui/resources.xml

```diff
@@ -210,15 +211,15 @@ Vocabulary
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="undo_cb"/>
         </object>
-        <accelerator key="z"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="Redo">
           <property name="icon_name">mypaint-redo-ltr-symbolic</property>
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="redo_cb"/>
         </object>
-        <accelerator key="y"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
On reflection, Ctrl+Y is common too - for example, in this very text input box.

Maybe GNOME are being a bit too perfectionist again. On the other claw, Ctrl+Y is overloaded for paste when in Emacs input mode. Whatever that is.

Maybe Ctrl+Shift+Z could be the fallback, Ctrl+Y could be the advertised Undo binding?